packages/backend-defaults/package.json

```diff
@@ -20,22 +20,55 @@
   "license": "Apache-2.0",
   "exports": {
     ".": "./src/index.ts",
+    "./cache": "./src/entrypoints/cache/index.ts",
```

Comment:
Does it make sense to create a codemod to assist adopters with updating their imports?

Reply:
I could try!